plugins/integrations/prometheus/src/main/java/org/apache/cloudstack/metrics/PrometheusExporterImpl.java

```diff
@@ -423,17 +524,23 @@ public String toMetricsString() {
         String zoneUuid;
         String state;
         int total;
+        String hosttags;
 
-        public ItemHost(final String zn, final String zu, final String st, int cnt) {
+        public ItemHost(final String zn, final String zu, final String st, int cnt, final String tags) {
             super("cloudstack_hosts_total");
             zoneName = zn;
             zoneUuid = zu;
             state = st;
             total = cnt;
+            hosttags = tags;
         }
 
         @Override
         public String toMetricsString() {
+            if (! Strings.isNullOrEmpty(hosttags)) {
```

Comment:
Can we use the `commons.lang3` library instead on these checks? cc. @GutoVeronezi

Reply:
@nvazquez thanks!

To contextualize, recently we finished a voting defining `commons.lang3` as the default string lib ([VOTE thread](https://lists.apache.org/thread.html/r806cd10b3de645c150e5e0e3d845c5a380a700197143f57f0834d758%40%3Cdev.cloudstack.apache.org%3E) & [RESULT thread](https://lists.apache.org/thread.html/r5898f717d506a0e160b9dc766b65769e2fe2f07f9cba70825843e272%40%3Cdev.cloudstack.apache.org%3E)). The code and the docs will be changed soon.

In this case, `!Strings.isNullOrEmpty(hosttags)` can be replaced with `StringUtils.isNotEmpty(hosttags)`.

Reply:
@GutoVeronezi and @nvazquez thanks!

I have refactored the code and used `commons.lang3` in [this commit](https://github.com/apache/cloudstack/pull/4438/commits/aaa1bcfa12732bd7069c31119b709a7b3416e931).

Reply:
@soreana, thanks!